.github/workflows/deployProd.yml

```diff
@@ -99,4 +96,41 @@ jobs:
           webhook_url: ${{ secrets.SR_ALERTS_SLACK_WEBHOOK_URL }}
           user_map: $${{ secrets.SR_ALERTS_GITHUB_SLACK_MAP }}
 
-# a post-prod health check workflow is defined in smokeTestDeployProd. See the Alert response wiki page for more details
\ No newline at end of file
+  failed_deploy_pagerduty_alert:
+    runs-on: ubuntu-latest
+    if: ${{ always() && needs.deploy.result != 'success' }}
```

Comment:
This would currently trigger the failed deploy alert if the `deploy` job gets cancelled or skipped. Should we opt for the higher sensitivity here of sending an alert if it is anything but successful? The alternative would be using `== 'failure` as used in other jobs which would not trigger the alert for a cancelled or skipped `deploy` job

Reply:
Think we can leave as is to get this work in and tune it down if it gets too noisy! Think this would be good to also flag any deploy issues since I know we've had some prod deploys fail silently us not know about them for a while, so this will hopefully catch those